index.html

```diff
@@ -154,21 +154,6 @@ <h3>Closure</h3>
       <section id="Team" class="featured_section">
         <h2>featured Speakers</h2>
         <ul class="teams"></ul>
```

Comment:
- [ ] (OPTIONAL) Good job so far 👍 Kindly add the `more` button to the mobile view of the homepage together with it's functionality for a better user interface 👍 
- Your design:
![Annotation 2022-04-22 040049](https://user-images.githubusercontent.com/85113567/164588478-eddfddb2-5025-4220-8492-6dbea9221127.jpg)
- Required design:
![Annotation 2022-04-22 040109](https://user-images.githubusercontent.com/85113567/164588502-8f9cc9a4-087c-4064-8f99-5b23bf5bf00e.jpg)